config/feature.js

```diff
@@ -15,6 +15,10 @@ function isEnabled(key) {
 }
 
 function isDevelopment() {
+  if(isFirefoxPanel()) {
+    // Default to production if compiling for the Firefox panel
+    return process.env.NODE_ENV === "development";
+  }
```

Comment:
Not sure I understand your question?

This is yet another case of the difference of build and run-time. In the firefox config we default NODE_ENV to be "production", but that doesn't happen at build-time. So we need to default to production at build-time too, but allow the user to override it.